src/components/ProjectUpdateForm.js

```diff
@@ -15,13 +15,51 @@ export default function ProjectUpdateForm({
   setEditing,
 }) {
   const [updatedData, setUpdatedData] = useState({ ...projectData })
+
+  const {
+    projectName,
+    pattern,
+    size,
+    nextStep,
+    image,
+    materialNeeds,
+    materialsExisting,
+  } = projectData
+
   const { register, errors, handleSubmit } = useForm({
     resolver: yupResolver(projectSchema),
-    defaultValues: { ...projectData },
+    defaultValues: {
+      projectName,
+      pattern,
+      size,
+      nextStep,
+      materialNeeds,
+      materialsExisting,
+    },
   })
 
+  const [loading, setLoading] = useState(false)
+  const [newImage, setNewImage] = useState('')
+
   return (
     <StyledForm onSubmit={handleSubmit(handleNewData)}>
+      <StyledImageUpload>
+        {image && <p>Change the image:</p>}
+        {!image && <p>Upload an image:</p>}
+        <InputField
+          labelText="Choose a file"
+          placeholder="upload an image"
+          type="file"
+          name="image"
+          registerFn={register}
+          onChange={uploadImage}
+        />
+      </StyledImageUpload>
+
+      {loading && <p className="loading">loading ...</p>}
+      {image && !newImage && <StyledThumbnail src={image} />}
```

Comment:
Hier bin ich mir grad unschlüssig wegen Ternary. Irgendwie schließt sich ja die Zeilen 60 + 61 gegenseitig aus, und in Zeile 48 hast du schon die Abfrage drin, was passiert wenn kein Image da ist.

Reply:
Ich hab hier ja eigentlich 3 mögliche Szenarien, es gibt ein Bild, es gibt ein neues Bild und es gibt kein Bild. Kann man das chainen? Da war ich mir nicht sicher... das wäre dann ja sowas wie das hier vielleicht, das könnte ich mal ausprobieren, was meinst du?
{image && !newImage ? <StyledThumbnail src={image}/> : newImage ? <StyledThumbnail src={newImage} /> : null}

Reply:
Ja, ich denke, dann ist deine Variante doch besser :) Naja, so hab ichdich nochmal zum Nachdenken gebracht 😁